prereise/gather/demanddata/transportation_electrification/data_process.py

```diff
@@ -0,0 +1,132 @@
+import pandas as pd
+
+#Download CSV of NHTS 2017 from https://nhts.ornl.gov/
+
+def data_filtering(census_division, filepath: str = "trippub.csv"):
```

Comment:
If we refactor this function to take a dataframe instead of a filepath, it will be easier to write automated tests which operate only on objects in-memory.